web/packages/teleterm/src/ui/ConnectMyComputer/DocumentConnectMyComputerSetup/DocumentConnectMyComputerSetup.tsx

```diff
@@ -35,9 +35,8 @@ interface DocumentConnectMyComputerSetupProps {
 export function DocumentConnectMyComputerSetup(
   props: DocumentConnectMyComputerSetupProps
 ) {
-  const [step, setStep] = useState<'information' | 'agent-setup'>(
-    'information'
-  );
+  const [step, setStep] =
+    useState<'information' | 'agent-setup'>('information');
```

Comment:
hmm.. i think these get updated when I run prettier

Reply:
This was fixed by #29352.